### ISSUE TITLE
Fine-Tuning approach

### ISSUE THREAD

**Issue by LorenzoBrugioni** [yerfor/SyntaSpeech#11]
Hi, I would like to use the pretrained model on LibriTTS to adapt the model on two target speakers for which I have about 40 minutes of training data each. 
Could you please share how would be the approach for fine tuning it? 
Any modules to freeze, decreasing learning rate, if it is actually possible in your opinion with that amount of data etc.. 
Any info would be useful.
Thanks for your work and have a good day.
